### PR TITLE
GitHub NuGet registry

### PR DESCRIPTION
**Pull request by MichielOda**

### CODE REVIEW

File: develop/TOOLS/NuGet/Producing_NuGet_GitHub.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Follow the instructions in [the official GitHub Docs](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions) to add a new secret:
+
+- **Name**: Enter `NUGETAPIKEY_GITHUB`.
+
+- **Secret**: Enter your personal access token.
+
+#### Creating a tag
+
+After creating the tag and running the workflow, you can find your package on the *Packages* page on [GitHub](https://github.com/orgs/SkylineCommunications/packages).
```
Comment (EdithLansens) [inline develop/TOOLS/NuGet/Producing_NuGet_GitHub.md:50]: Should we here not provide more information on how to create a tag first, seeing as how the title of this section is "Creating a tag", @MichielOda?

Comment (MichielOda) [reply develop/TOOLS/NuGet/Producing_NuGet_GitHub.md:50]: Creating a tag can be done in multiple ways and at this point the expectation is that the user at least know how git works as this is a base requirement.
I think a better title would be: 'Publishing the package'.

Comment (EdithLansens) [reply develop/TOOLS/NuGet/Producing_NuGet_GitHub.md:50]: I have updated the title!

**Comment by EdithLansens** [comment]
Let me know when this pull request is ready to merge, @MichielOda!

**Comment by MichielOda** [comment]
> Let me know when this pull request is ready to merge, @MichielOda!

Good for me!

**Pull request merged**